### DISCUSSION TITLE
Remembering

### DISCUSSION THREAD

**Discussion by VParth2005** [Cyfrin/foundry-full-course-cu#1258] [General]
Hey, I just started learning solidity in the second chapter of the video. I am now wondering if I should take notes while watching the video (if yes, how?) should I remember the syntax as well?

**Comment by Vishal-coding** [comment]
Yes, You don't need to urge in order to remember syntax in the beginning. Better you can take notes in the order you can easily understand and once you start code again and again in practice those will be remembered easily. Coding continuously will improve your remembrance my friend that's how i am doing now.

**Reply by VParth2005** [reply]
So from where would you suggest learning to code solidity?
I am also having a hard time understanding what is happening in the "FundMe" section of the course. What can I do about it?

**Reply by Vishal-coding** [reply]
Brother, I know it is hard but i know Patrick's sir teaching solidity is the most effective as well as easy. But i would suggest that even if you could not understand , watch those videos as much again and implement with your own variable declaration. So that you could be able to know the flow and while coding help yourself with solidity docs which will be very useful. I am not a big guy in this even though I had not even started the foundry, this way really helps me. Take breaks and make progress.I wish you to have a great journey. 👍👍

Even though you find issues feel free to post your coding doubts in a format and the community will support you.

**Reply by Vishal-coding** [reply]
You can also learn solidity from YouTube channels like

1.Dapp University
2.Eat the blocks
